examples/slash-commands/cogs/groupcog.py

```diff
@@ -19,22 +19,102 @@ def __init__(self, bot: CustomBot) -> None:
 
     @group.command(name="pow", description="Raise a number to a power")
     async def power(self, inter: discord.Interaction, a: int, b: int) -> None:
+        """
+        Raise a number to a power.
+
+        Parameters
+        ----------
+        inter : discord.Interaction
+            The interaction.
```

Comment:
what's the point in documenting this?

Reply:
Just ocd 💀 ide gives a yellow line with param not documented